docs/src/algebraic.md

```diff
@@ -10,23 +10,13 @@ end
 Nemo allows working with exact real and complex algebraic numbers.
 
 The default algebraic number type in Nemo is provided by Calcium. The
-associated field of algebraic numbers is represented by the constant
-parent object called `CalciumQQBar`.
+associated field of algebraic numbers can be constructed using
+`QQBar = algebraic_closure(QQ)`. We will leave out this line from
+all code blocks on this page for brevity.
 
-For convenience we define
-
-```
-QQBar = CalciumQQBar
-```
-
-so that algebraic numbers can be constructed using `QQBar` instead of
-`CalciumQQBar`. Note that this is the name of a specific parent object,
-not the name of its type.
-
-
- Library        | Element type  | Parent type
-----------------|---------------|--------------------
-Calcium         | `QQBarFieldElem`       | `QQBarField`
+ Library        | Element type     | Parent type
+----------------|------------------|--------------------
+Calcium         | `QQBarFieldElem` | `QQBarField`
```

Comment:
Again not part of this PR, but should we really still refer to "Calcium" here (and arb elsewhere), given that they have all been merged into FLINT?